api/src/pcapi/core/mails/backends/sendinblue.py

```diff
@@ -78,13 +78,17 @@ def send_mail(
 
     def _get_whitelisted_recipients(self, recipient_list: Iterable) -> list:
         whitelisted_recipients = set()
+        dev_alias_email_address_split = settings.DEV_EMAIL_ALIAS_ADDRESS.split('@')
+        print(dev_alias_email_address_split)
         for recipient in recipient_list:
             # Imported test users are whitelisted (Internal users, Bug Bounty, audit, etc.)
             user = find_user_by_email(recipient)
-            qatest_recipient = recipient.startswith("qa-test") and recipient.endswith("@passculture.app")
+            qatest_recipient = recipient.startswith(dev_alias_email_address_split[0]) and recipient.endswith(f"@{dev_alias_email_address_split[1]}")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏`recipient.endswith(dev_alias_email_address_split[1])` suffit ici je pense

Reply:
C'est plus vrai avec l'arobase que sans